exercises/practice/twelve-days/twelve_days.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <string>
+
+namespace twelve_days {
+
+std::string verse(int day);
+std::string lyrics(int start_day, int end_day);
+std::string lyrics(int end_day);
```

Comment:
Similarly to the previous comment, those should be removed. It's up to the student to derive an appropriate function signature from the tests.